src/shared/loop-util.c

```diff
@@ -817,16 +817,24 @@ static LoopDevice* loop_device_free(LoopDevice *d) {
         }
 
         /* Now that the block device is released, let's also try to remove it */
-        if (control >= 0)
-                for (unsigned n_attempts = 0;;) {
+        if (control >= 0) {
+                useconds_t delay = 50 * USEC_PER_MSEC;
```

Comment:
useconds_t -> usec_t

Reply:
This is used to call `usleep()`, which takes `useconds_t`.

Reply:
I am pretty sure we should stick to usec_t unvirsally, for all our internal calculations and at best convert to useconds_t and similar when we actually have to, given that the bit width actually *does* matter a lot for µs values.

Reply:
But the *only* use of this variable is to grow the value and to pass it to `usleep(3)`. So using a wider type would only make it more likely that we get confused. And we're not doing any "internal calculations" that could make use a wider type: the final result is as wide as any internal calculation.